tests/acceptance/expected-failures-API-on-OCIS-storage.md

```diff
@@ -624,12 +624,6 @@ Not everything needs to be implemented for ocis. While the oc10 testsuite covers
 - [coreApiWebdavOperations/listFiles.feature:394](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/coreApiWebdavOperations/listFiles.feature#L394)
 - [coreApiWebdavOperations/listFiles.feature:399](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/coreApiWebdavOperations/listFiles.feature#L399)
 
-### [graph/users: enable/disable users](https://github.com/owncloud/ocis/issues/3064)
-
```

Comment:
these tests fail in stable because of this https://github.com/owncloud/ocis/issues/7087 but if the fix won't be backported to stable we can also skip these tests in stable. waiting for some decision on what to do